src/components/Breadcrumbs/index.tsx

```diff
@@ -1,21 +1,30 @@
-import useUIStore from '../../hooks/useUIStore';
 import MuiBreadcrumbs, { BreadcrumbsProps } from '@mui/material/Breadcrumbs';
 import Link from '@mui/material/Link';
 import Typography from '@mui/material/Typography';
 import { observer } from 'mobx-react-lite';
 import React from 'react';
+import useUIStore from '../../hooks/useUIStore';
 
 function Breadcrumbs(props: BreadcrumbsProps) {
   const uiStore = useUIStore();
 
   return (
     <MuiBreadcrumbs {...props}>
-      {uiStore.breadcrumb.paths.map((path) => (
+      {uiStore.breadcrumb.linkedPaths.map((path) => (
         <Link
           key={path.link}
           underline="hover"
           color="inherit"
           href={path.link}
+          onClick={(e) => {
```

Comment:
We could separate this logic from the JSX by creating a function that return a function

Example: 
```ts
function handleLinkClick(path: PathType) {
  return (e) => {
    e.preventDefault();
    if (path.onClick && !path.link) {
      path.onClick();
    }
  }  
}

onClick={handleLinkClick(path)}
```